contributor-guide/modules/ROOT/pages/version-control.adoc

```diff
@@ -1,4 +1,57 @@
-= Version Control (Git)
+= Version Control
 :navtitle: Version Control
 
-*Content under construction.*
+Boost uses the Git version control system for its development. This allows multiple contributors to work on the libraries concurrently, while keeping track of the changes that each person makes. Git is a distributed version control system, which means each contributor has their own local copy of the entire project, including its history.
```

Comment:
I don't know if placing this under "Releases" is the best choice. It took me some time to find it. I don't know if it fits in any of the current categories, but feels weird under releases, since you use it for regular development.

Reply:
@anarthal Yes, perhaps a new menu entry "Development" would work, adding this and other programming/development topics?